test/test_jit.py

```diff
@@ -1480,6 +1482,28 @@ def test_quant_fusion(self):
             torch._C._jit_pass_quant_fusion(graph)
             FileCheck().run(input_str, graph)
 
+    def test_quantized_conv_relu_fusion(self):
```

Comment:
@raghuramank100 the new test is going to test the fusion since we do inline etc. in convert script